src/main/java/org/jenkinsci/plugins/codesonar/models/CodeSonarAnalysisData.java

```diff
@@ -0,0 +1,148 @@
+package org.jenkinsci.plugins.codesonar.models;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jenkinsci.plugins.codesonar.CodeSonarAlertCounter;
+import org.jenkinsci.plugins.codesonar.models.analysis.Analysis;
+import org.jenkinsci.plugins.codesonar.models.json.CodeSonarAnalysisWarningCount;
+import org.jenkinsci.plugins.codesonar.models.metrics.Metrics;
+import org.jenkinsci.plugins.codesonar.models.procedures.Procedures;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarAnalysisData {
```

Comment:
Is this `CodeSonarAnalysisData` class needed?  Don't we manage data in the "data loader" class now?